src/cudamatrix/cu-kernels.cu

```diff
@@ -1763,6 +1763,31 @@ static void _group_max(Real *y, const Real *x, MatrixDim d, int src_stride,
   }
 }
 
+template<typename Real>
+__global__
+static void _parametric_relu(Real*y, const Real*x, MatrixDim d, int src_stride, const Real*a, const Real*b) {
```

Comment:
Hi, I'll be highlighting the code, pointing out at some details od the coding style we use.
The basic rule is that the style of the code should be the same as elsewhere in the file, 
including the white-spaces: 
'Real*y' -> 'Real *y'
'j*d.stride' -> 'j * d.stride'

(and also there should be no whitespace at the end of the lines)